src/tests/ftest/launch.py

```diff
@@ -953,6 +958,161 @@ def rename_logs(avocado_logs_dir, test_file):
                 test_logs_dir, new_test_logs_dir, error))
 
 
+USE_DEBUGINFO_INSTALL = True
+
+def install_debuginfos():
+    """Install debuginfo packages"""
+
+    install_pkgs = [{'name': 'gdb'}, {'name': 'python-magic'}]
+    cmds = []
+
+    if USE_DEBUGINFO_INSTALL:
+        cmds.append("sudo debuginfo-install -y "                   \
+                    "--exclude ompi-debuginfo,gcc-debuginfo,"      \
+                               "gcc-base-debuginfo "               \
+                    "daos-server cart libpmemobj python openmpi3")
+    else:
+        import yum
```

Comment:
(pylint-import-error) Unable to import 'yum'

Reply:
Doesn't exist for Python 3.